resources/shaders/deferred_composition.hlsl

```diff
@@ -75,15 +79,15 @@ void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
 		float3 sampled_environment_map = pref_env_map.SampleLevel(linear_sampler, reflect(-V, normal), roughness * MAX_REFLECTION_LOD);
 		
 		// Get irradiance
-		float irradiance = lerp(
+		float3 irradiance = lerp(
 			irradiance_map.SampleLevel(linear_sampler, flipped_N, 0).xyz,
-			screen_space_irradiance[screen_coord].xyz,
+			screen_space_irradiance.SampleLevel(linear_sampler, uv, 0).xyz,
```

Comment:
Again why sampling and you don't want to do linear here anyway. see others as well.